include/nbl/video/IPhysicalDevice.h

```diff
@@ -512,11 +512,42 @@ class NBL_API2 IPhysicalDevice : public core::Interface, public core::Unmovable
             }
         }
 
+        
+        static inline uint32_t getMaxInvocationsPerComputeUnitsFromDriverID(E_DRIVER_ID driverID)
+        {
+            const bool isIntelGPU = (driverID == E_DRIVER_ID::EDI_INTEL_OPEN_SOURCE_MESA || driverID == E_DRIVER_ID::EDI_INTEL_PROPRIETARY_WINDOWS);
+            const bool isAMDGPU = (driverID == E_DRIVER_ID::EDI_AMD_OPEN_SOURCE || driverID == E_DRIVER_ID::EDI_AMD_PROPRIETARY);
+            const bool isNVIDIAGPU = (driverID == E_DRIVER_ID::EDI_NVIDIA_PROPRIETARY);
+            if (isNVIDIAGPU)
+                return 32u * 48u; // RTX 3090 (32 Threads/Warp * 48 Warp/SM)
```

Comment:
yeah basically on NV, its either 1024, 1536 or 2048 depending on the architecture, but latest GPUs have 1536, so this is correct.